_data/simple-icons.json

```diff
@@ -1000,6 +1000,11 @@
             "hex": "CC2929",
             "source": "https://support.humblebundle.com/hc/en-us/articles/202742060-Bundle-Logos"
         },
+        {
+            "title": "Icloud",
+            "hex": "297DED",
```

Comment:
I'm getting in the range of `#3693f3` to `#93dcf7` when getting the color from the official logo - the only place that's as dark as `#297ded` is the stroke around the edge of the bubbles (which I don't think is representative of the brand).

Mind including how you found the color?